janfix/extension-geogebra
# Fix/ADF-1488/Load error with TAO Advance

### Related to: https://oat-sa.atlassian.net/browse/ADF-1488

### Summary

The PCI failed to load in TAO Advance. This is due to a wrong URI used to load an internal resource.

### Details

URI to shared resources offered by TAO for developing a PCI must be scoped to `taoQtiItem/portableLib/`.

Also added a quick fix for an issue occurring when the response is restored. Typically, this happens when revisiting an item containing the PCI. I've added a TODO for properly addressing the case. I think the `setResponse()` was hijacked for another meaning than the QTI context.

### How to test

- Install the PCI
- add it to an item
- check the preview using TAO Advance
- add the item to a test
- publish the test to TAO Advance
- take the test
